src/app/services/ServiceContainer.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Button from '../../components/button';
+import NavBar from '../NavBar';
+import ServicesInfoPage from './ServicesInfoPage';
+import ServicesRecapPage from './ServicesRecapPage';
+
+const data = {
+  shelter: {
+    1: { name: 'Community shelter' },
+  },
+  food: {
+    1: { name: 'Soup Kitchen' },
+    2: { name: 'Mobile Soup Kitchen' },
+    3: { name: 'Food Panty' },
+    4: { name: 'Brown Bag Lunch' },
+  },
+  other: {
+    1: { name: 'Project ID' },
+  },
+};
+
+const PAGES = {
+  INFO: 'Services info',
+  RECAP: 'Services recap',
+};
```

Comment:
This stuff should also probably go into the route so that when the user refreshes the page, they can continue in their existing flow.

Reply:
The location flow can probably be used as a reference for this.

Reply:
Following up from our offline conversation, we decided to use the server to persistent the data between these screens. I'll update this branch to mock these endpoints until we have them.